### ISSUE TITLE
Display mocked data for the facility attributes not stored in the backend

### ISSUE THREAD

**Issue by shahraz1998** [amosproj/amos2024ss01-xcelerator-demo-app#156]
## User story

1. As a facility developer
2. I want / need to have heterogeneous data that includes mocked information
3. So that the demo application properly displays different facility scenarios

## Acceptance criteria

- [ ] Facilities should not all be green and look like they were before the backend frontend connection.
- [ ] Take the assetId and pump/envData from the backend, but always show our mocked names, subheading, ..
- [ ] Add a "seed the database" button and make it show if there are no facilities in the database (pressing it will seed the database and add the facilities).
- [ ] Adjust real and mocked pump data.
- [ ] Only "real" Pumps are 002 and 010. 

## DoD general criteria

- [ ] Feature has been fully implemented
- [ ] Feature has been merged into the mainline
- [ ] All acceptance criteria were met
- [ ] Product owner approved features
- [ ] All tests are passing
- [ ] Developers agreed to release

**Comment by Sungonogi** [comment]
After talking with Igor on the day at Siemens yesterday we decided that all but the last acceptance criteria are good. For the last one we decided now what excactly to do:
1. We only show the actual chart data (which calls the Siemens API) from
- PUMP-002 (id: ba9361db82f4411297bdd8494ecb040f) which is not pumping properly all the time -> yellow status
- Playground-MSAgentPump (id: 8ccb6c1c6c464e5784410e5a1c8a8def) which is working fine -> green status

2. We store some mocked up data for other facilities in our backend for the charts. This is because the application should also work without an internet connection, which is necessary to call the Siemens API for their data. So we might have to wait for the backend to be ready for that
